src/facade/reply_builder.cc

```diff
@@ -360,6 +360,17 @@ void RedisReplyBuilder::SendScoredArray(const std::vector<std::pair<std::string,
   }
 }
 
+void RedisReplyBuilder::SendNumSubArray(const std::vector<std::pair<std::string, long>>& arr) {
```

Comment:
`ReplyBuilder` isn't the best place to place this logic. It's a protocol wrapper, providing building blocks for sending replies.

Reply:
fixed: https://github.com/dragonflydb/dragonfly/pull/2282/commits/4078b3426e136329e17ac3d80eea8baf931ffb45